src/multibody/data.cpp

```diff
@@ -0,0 +1,5 @@
+#include "data.hpp"
+
+#if PINOCCHIO_ENABLE_TEMPLATE_INSTANTIATION
+template class pinocchio::DataTpl<double>;
```

Comment:
We should also generate some methods.

Reply:
It isn't don't automatically ?

Reply:
Not for the templated methods.

Reply:
Now that I think about it, it is obvious. How would the compiler know what type to use !